main.go

```diff
@@ -110,11 +110,6 @@ func main() {
 		return
 	}
 
-	if strings.HasPrefix(kongCtx.Command(), format.SetOrgCommand) {
```

Comment:
Why did we actually remove this?

Reply:
this block of commands was design to:

> handle the login/oidc cmds separately as we should not try to get the API client if we're not logged in.

The previous version of `set-org` command performed the login operation for us. But we don't need this anymore. We just use a normal `Run()` with `client *api.Client` as argument. And we need `client` to read kubeconfig path or eventually (if org argument is missing) forward the request to `whoami` command which also needs `client`.